### PR TITLE
Naming changes for consistency across SDKs

### PR DESCRIPTION
**Pull request by karishma-chawla**
# Description

Updated API and parameter names for consistency across SDKs.

## Issue reference

https://github.com/dapr/python-sdk/issues/150
https://github.com/dapr/dapr/issues/2535

Please reference the issue this PR will close: #150

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Comment by karishma-chawla** [comment]
Added changes to tox.ini file by mistake, reverting them.

File: dapr/actor/runtime/_reminder_data.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def as_dict(self) -> Dict[str, Any]:
         if self._state is not None:
             encoded_state = base64.b64encode(self._state)
         return {
-            'name': self._name,
+            'reminderName': self._reminder_name,
```
Comment (mukundansundar) [inline dapr/actor/runtime/_reminder_data.py:75]: @yaron2 @youngbupark Similar to this should `timerName` be returned in `as_dict` call for the timer class seen below?
See line 79 in `_timer_data.py` file.

File: dapr/clients/grpc/_response.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def get_headers(self, as_dict: bool = False) -> Union[MetadataDict, MetadataTupl
 
 
 class InvokeServiceResponse(DaprResponse):
```
Comment (mukundansundar) [inline dapr/clients/grpc/_response.py:68]: @yaron2 Should this be renamed to InvokeMethodResponse? and similar changes for other classes?

File: dapr/proto/common/v1/common_pb2_grpc.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
```
Comment (mukundansundar) [inline dapr/proto/common/v1/common_pb2_grpc.py:2]: @karishma-chawla was this automatically added?

File: dapr/clients/grpc/_response.py (hunk 2 above)
Comment (karishma-chawla) [reply dapr/clients/grpc/_response.py:68]: I actually did rename to InvokeMethodResponse/Request, only to realize while testing that we have defined them as InvokeService* in Dapr runtime https://github.com/dapr/java-sdk/blob/6a44d51279d7d6d5ccdfd63346b4c83d00e7eeb8/sdk/src/main/java/io/dapr/client/DaprClient.java, so ended up changing it back.

Jave SDK has also similar naming convention right now https://github.com/dapr/java-sdk/blob/9dcff43ef08497b7169b93207002f450a76236bd/sdk/src/main/java/io/dapr/client/domain/InvokeServiceRequest.java

I do think though that we should rename everything including runtime to InvokeMethod* for consistency, but not sure if it should be prioritized over other things.

File: dapr/proto/common/v1/common_pb2_grpc.py (hunk 3 above)
Comment (karishma-chawla) [reply dapr/proto/common/v1/common_pb2_grpc.py:2]: I ran the script in the repo for generated code to let it pick latest changes https://github.com/dapr/python-sdk/blob/master/tools/regen_grpcclient.sh

File: dapr/clients/grpc/_response.py (hunk 2 above)
Comment (karishma-chawla) [reply dapr/clients/grpc/_response.py]: I noticed that Artur updated Java SDK now to use InvokeMethodResponse/Request, so I've updated the PR to reflect that in python SDK as well. Calls to runtime are still InvokeServiceResponse as I mentioned in the previous comment, just a heads up.

**Review by artursouza — APPROVED** [review]

**Pull request merged**